### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by ragangithub**
- Added a new column called species
- Added new animals into table
- Used transactions to update table
Screenshot (271)
Screenshot (272)
Screenshot (273)
Screenshot (274)
Screenshot (275)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.3;
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species from animals; -- verify that change was made
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql:16]: * Kindly  verify that the change was undone by adding this query ``SELECT species from animals;``

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @ragangithub 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Data modified properly 👍 
* Well written PR 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- verified that the change was undone by adding this query SELECT species from animals;
Screenshot (271)

**Review by Reem-lab — APPROVED** [review]
# Hi @ragangithub,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  No Linters errors 🔥
- [ ]  Great job working on the functionality💯
- [ ]  Followed GitFlow ⭐🚀


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**